pallets/bridge-mapping/src/lib.rs

```diff
@@ -65,6 +65,15 @@ pub mod pallet {
 		/// Admin is able to set/remove resource mappings.
 		type AdminOrigin: EnsureOrigin<Self::Origin>;
 
+		/// Resource id type
+		type ResourceId: Member
```

Comment:
Generalizing here, we can avoid the tight coupling with `pallet_nft`, avoiding the waterfall dependency that this implies.